docs/metasploit-framework.wiki/Metasploit-Guide-HTTP.md

```diff
@@ -159,3 +159,30 @@ Module advanced options (auxiliary/scanner/http/title):
    VERBOSE               false                                              no        Enable detailed status messages
    WORKSPACE                                                                no        Specify the workspace for this module
 ```
+
+### HTTP Multiple-Headers
+Additional headers can be set via the ```HTTPRawHeaders``` option.
```

Comment:
When the content is not split across multiple lines you can use a single backquote.
```suggestion
Additional headers can be set via the `HTTPRawHeaders` option.
```